wiki/FitNesseRoot/HsacExamples/SlimTests/HttpTests/content.txt

```diff
@@ -44,10 +44,11 @@ The main commands/keywords (i.e. public methods) offered by !-nl.hsac.fitnesse.f
 The commands/keywords (i.e. public methods) added by !-nl.hsac.fitnesse.fixture.slim.JsonHttpTest-! are listed below.
 
 -|Comment|
-|post values as json to <url>      |Sends a POST containing all values (as JSON properties) to the url.                                                          |
-|json path <expression>            |Evaluates the (definite) expression against the last response received.                                                      |
-|json path count <expression>      |Evaluates the (indefinite) expression against the last response received, counting the number of results.                    |
-|all json path matches <expression>|Evaluates the expression against the last response received and returns all results. Intended to be used with 'show' command.|
+|post values as json to <url>                   |Sends a POST containing all values (as JSON properties) to the url.                                                                            |
+|json path <expression>                         |Evaluates the (definite) expression against the last response received.                                                                        |
+|json path count <expression>                   |Evaluates the (indefinite) expression against the last response received, counting the number of results.                                      |
+|all json path matches <expression>             |Evaluates the expression against the last response received and returns all results. Intended to be used with 'show' command.                  |
+|set json path <expression> to <value> in <json>|Evaluates the expression against the provided json and -if exactly one key is found- replaces that key's existing value with the provided value|
```

Comment:
I considered the ordering to be the other way around but didn't, because a large json string would make the test log less readable when the value to set would appear last. 

Your suggestion to update the current response would void that argument so I guess that's a good idea. I'll update tonight. Should we remove the method that allows a custom string to be sent as the json to update? Can't really think of a proper use case for that, other than creating test data templates (which we can already achieve by using ftl's.)